lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -1515,6 +1515,19 @@ mrs_realloc(void *ptr, size_t size)
 	mrs_debug_printf("mrs_realloc: called ptr %p ptr size %zu new size %zu\n",
```

Comment:
A few typos in the commit log: s/performes/performs/, s/the frees/then frees/, "but for asymptotically" seems to be missing a word ("asymptotic cases" perhaps?), s/4th allocating/4th allocation/, s/entierly/entirely/, s/unnecessicary/unnecessary/, s/improveing/improving/.